### PR TITLE
:sparkles: Feature: add some options in slides configuration

### LINKED ISSUE #298: Problem with auto-Slide configuarion
## I want to create a new issue

- [x] Yes, I have read [FAQ](https://github.com/Molunerfinn/hexo-theme-melody/blob/dev/FAQ.md).
- [x] Yes, I have read [Hexo Docs page](https://hexo.io/docs/), especially [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [x] Yes, I have read [Hexo-theme-melody Documentation](https://molunerfinn.com/hexo-theme-melody-doc/).
- [x] And yes, I already searched for current [issues](https://github.com/Molunerfinn/hexo-theme-melody/issues?utf8=%E2%9C%93&q=is%3Aissue) and this did not help me.

## Melody Information

**Melody Version:** version 1.8.2

**System:**
    OS: OS X El Capitan 10.11.6
**Binaries:**
    Node: 14.8.0 - /var/folders/0y/f7c873891kz5d_s7qr95vj740000gn/T/yarn--1601029197092-0.1531156558919038/node
    Yarn: 1.22.5 - /var/folders/0y/f7c873891kz5d_s7qr95vj740000gn/T/yarn--1601029197092-0.1531156558919038/yarn
    npm: 6.14.7 - /usr/local/bin/npm
**Browser:** Opera Desktop for Mac

## Expected behavior

The presentation will step through slides automatically

## Actual behavior

image

My presentation doesn't step through slides automatically

## Steps to reproduce the behavior
```md
---
title: Biology
date: 2020-09-10 19:57:52
tags: Biology
layout: slides
slide:
  loop: true
  autoSlide: 5000
---

<!-- .slide: data-background="#49B1F5" -->

My content
```

## Other information

* [My blog](https://loctran016.gitlab.io)
* [My source code](https://gitlab.com/LocTran016/loctran016.gitlab.io)

### PR DESCRIPTION
**Pull request by LocTran016**
Loop, autoSlide, and autoSlideStoppable are working

image

Closed #298

### CODE REVIEW

File: layout/includes/slide/script.pug [hunk 1]
```diff
@@ -10,16 +10,22 @@ script(src=url_for("https://cdn.jsdelivr.net/npm/reveal.js@3.6.0/js/reveal.min.j
...
 - var parallaxBackgroundHorizontal = slide.parallaxBackgroundHorizontal || defaultSlide.parallaxBackgroundHorizontal
 - var parallaxBackgroundVertical = slide.parallaxBackgroundVertical || defaultSlide.parallaxBackgroundVertical
+- var autoSlide = slide.autoSlide || defaultSlide.autoSlide
+- var loop = slide.loop || defaultSlide.loop
+- var autoSlideStoppable = slide.autoSlideStoppable || defaultSlide.autoSlideStoppable
 
 script.
   Reveal.initialize({
-    mouseWheel: !{mouseWheel},
+    mouseWheel: '!{mouseWheel}',
```
Comment (Molunerfinn) [inline layout/includes/slide/script.pug:19]: `mouseWheel` is a boolean value, don't add `'`

**Review by Molunerfinn — CHANGES_REQUESTED** [review]

File: layout/includes/slide/script.pug (hunk 1 above)
Comment (LocTran016) [reply layout/includes/slide/script.pug:19]: Oh, sorry.

Comment (LocTran016) [reply layout/includes/slide/script.pug]: Fixed below

Comment (Molunerfinn) [reply layout/includes/slide/script.pug]: `autoSlide` is `number` \ `loop` is `boolean`, so don't add `'` also

**Pull request merged**